### ISSUE TITLE
.Clocks app doesn't pick up time after system reboot

### ISSUE THREAD

**Issue by mikesugg** [hereafter/powergadgets#3]
I installed the .Clocks app on 4 Win 10 Pro computers (with all patches up to date).  After rebooting the devices, the digital clock comes up and says 0:00 AM and doesn't update.  To force the update, I have to right click on the clock, selection options, and then close options and then the time updates.

How do I get this to update automatically after reboot?

2nd issue--instead of showing the time as 12:21 PM today, the clock showed 0:21.  

![reboot clocks img](https://user-images.githubusercontent.com/77711211/105124054-46df2700-5aa7-11eb-8ef1-352d2df02ef4.JPG)

**Comment by hereafter** [comment]
Yes, recent system updates triggers all frozen upon startup. please toggle between show settings and close settings to get the clock running again.

I am at a process of re-writing a memory efficient engine. Please bare with this issue first.
